src/app/modules/recipe-create/services/recipe-create.service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable, throwError } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+import {Payload} from "../models/recipe-create.model";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class RecipeCreateService {
+  //TODO: change to actual url when recipe api will be created
+  url = 'http://localhost:3000/recipe_POST';
+  constructor(private http: HttpClient) { }
+
+  create(payload: Payload): Observable<Payload[]> {
+    return this.http.post<Payload[]>(this.url, payload)
+      .pipe(catchError(this.handleError));
+  }
+
+  private handleError(error: Response) {
+    if (error.status === 400)
+      return throwError(alert('Bad request.'));
```

Comment:
always use `{` `}` with `if`s
```suggestion
    if (error.status === 400) {
      return throwError(alert('Bad request.'));
    }
```